### ISSUE TITLE
Some examples not working on webpage

### ISSUE THREAD

**Issue by TheLordWolf** [raysan5/raylib.com#81]
### Some examples showed in the "Example" page are broken.
I found that the following examples are broken on the website.
I opened the development console to check the logs of Raylib for each examples, and i added the log file + a screenshot of each examples in this issue.

(i tried the examples with Firefox 129.0.2 64bits on Kubuntu 23.10 and i also tried with my Android smartphone with Firefox 130.0.)
EDIT i've also tried the examples on Windows 10 with Firefox (130.0 64 bits), Brave (1.70.117 64 bits) and Edge (128.0.2739.79 64 bits). They exemple are also broken. 

(in the screenshots i've the website in "dark mode" i simply used an extension called Dark Reader.
I tried to deactivated it, but even deactivated the examples were still broken.)

### Broken  example
------
**1. Deferred Render (Shader example category) Draws a full black screen (the FPS aren't drawn).** 
------
[deferred_render_log.txt](https://github.com/user-attachments/files/17074171/deferred_render_log.txt)

![deferred_render](https://github.com/user-attachments/assets/7f03786a-716a-45fa-9326-50e69c34a58a)

------
**2. Mesh Instancing (Shader example category) Draws a white screen.**
------
[mesh_instancing_log.txt](https://github.com/user-attachments/files/17074168/mesh_instancing_log.txt)

![mesh_instancing](https://github.com/user-attachments/assets/ade6ee13-ef42-40e8-8e98-7af6831f71c2)

------
**3. Lightmap (Shader example category) Draws the model but there's no lightmap.**
------

[lightmap_log.txt](https://github.com/user-attachments/files/17074165/lightmap_log.txt)

![lightmap](https://github.com/user-attachments/assets/813c7fb1-9b29-4a06-ac75-4723883cb77f)

------
**4. Julia Set (Shader example category) Draws the texts but not the fractal.**
------
[julia_set_log.txt](https://github.com/user-attachments/files/17074162/julia_set_log.txt)

![julia_set](https://github.com/user-attachments/assets/5d60c113-941a-4881-8d84-cfbb00354575)

------
**5. SVG Loading (Texture example category) Draws the text but without the SVG texture.**
------

[svg_loading_log.txt](https://github.com/user-attachments/files/17074155/svg_loading_log.txt)
![svg_loading](https://github.com/user-attachments/assets/634e16ce-5472-44cb-b68c-33ec55e815f6)

------
**6. Skybox (Model example category) Draws everything correctly but without the skybox.**
------

[skybox_log.txt](https://github.com/user-attachments/files/17074147/skybox_log.txt)
![skybox](https://github.com/user-attachments/assets/a18760bf-5499-4567-b2a7-598436cb17f2)

------
**7. Loading Thread (Core example category) Doesn't "load".**
------

_Nothing has been logged into the console_
![loading_thread](https://github.com/user-attachments/assets/3c9dcedc-4e14-47b0-ae88-3f18fd868b9b)

To summarize the logs.
For the SVG Loading and Skybox loading examples, the file formats aren't supported. 
For the Lightmap example, the Shader failed to load.
And for the other examples, the Shaders failed to compile.

But i don't know why the Loading Thread example doesn't "load" at all.

**Comment by raysan5** [comment]
I'm closing because it belongs to old raylib release... I'm aware of the issues...
